components/bl-mapbox-component/src/helpers.js

```diff
@@ -243,21 +253,35 @@ const addPolygons = (mapRef, polygonsArray, onPolygonClick, map) => {
       },
     });
 
+    if (polygonOutline) {
+      map.addLayer({
+        id    : `${ id }-outline`,
+        type  : 'line',
+        source: id,
+        layout: {},
+        paint : {
+          'line-color': polygonOutlineColor,
+          'line-width': polygonOutlineWidth
+        }
+      });
```

Comment:
MapController class could have a method that hide this logic from the component.
Here it could looks like

`map.addPolygonOutline(id, polygonOutlineColor,polygonOutlineWidth)`